src/VCR/Storage/AbstractStorage.php

```diff
@@ -54,10 +54,12 @@
      * If the cassetteName contains PATH_SEPARATORs, subfolders of the
      * cassettePath are autocreated when not existing.
      *
-     * @param string $cassettePath Path to the cassette directory.
-     * @param string $cassetteName Path to the cassette file, relative to the path.
+     * @param string  $cassettePath   Path to the cassette directory.
+     * @param string  $cassetteName   Path to the cassette file, relative to the path.
+     * @param string  $defaultContent Default data for this cassette if its not existing
+     * @param boolean $writeable      Whether the file must be writeable
      */
-    public function __construct($cassettePath, $cassetteName, $defaultContent = '[]')
+    public function __construct($cassettePath, $cassetteName, $defaultContent = '[]', $writeable = true)
```

Comment:
i was pondering instead allowing to pass `false` for $defaultContent to indicate that this is readonly. as defaultContent makes no sense for readonly anyways. but that is a bit cryptic - on the other hand we now need to pass a default in the factory to get to the 4th parameter... what do you prefer? i tend to change it to say $defaultContent false means its readonly.